### PR TITLE
8.6.1 CHANGES entries.

### PR DESCRIPTION
**Pull request by maximedenes**
Any feedback? Missing entries?

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -1,3 +1,55 @@
...
+- Fix bug #5486, don't reverse ids in tuples 
+- Fixing #5522 (anomaly with free vars of pat)
+- Fix bug #5526, don't check for nonlinearity in notation if printing only
+- Fix bug #5255
+- Fix bug #3659: -time should understand multibyte encodings.
+- FIx bug #5300: Anomaly: Uncaught exception Not_found" in "Print Assumptions".
+- Fix outdated description in RefMan.
+- Repairing `Set Rewriting Schemes`
+- Fixing #5487 (v8.5 regression on ltac-matching expressions with evars).
+- remove unneeded -emacs flag in coq-prog-args in test-suite files
```
Comment (Zimmi48) [inline CHANGES]: Possibly non relevant for this list.

File: CHANGES [hunk 2]
```diff
@@ -1,3 +1,55 @@
...
+- Instances should obey universe binders even when defined by tactics.
+- Intern names bound in match patterns
+- funind: Ignore missing info for current function
+- Do not typecheck twice the type of opaque constants.
+- show unused intro pattern warning
+- [future] Be eager when "chaining" already resolved future values.
+- Opaque side effects
+- Fix #5132: coq_makefile generates incorrect install goal 
+- Run non-tactic comands without resilient_command
+- [ltac] Move dummy plugin to plugins folder.
```
Comment (Zimmi48) [inline CHANGES]: Same remark.

**Review by Zimmi48 — COMMENTED** [review]
You ask for feedback so I find something to say, but that's very minor.

File: CHANGES [hunk 3]
```diff
@@ -1,3 +1,55 @@
+Changes from 8.6 to 8.6.1
+=========================
+
+- Fix #5380: Default colors for CoqIDE are actually applied.
+- Fix plugin warnings
+- Use more neutral wording instead of mentioning CoqIDE
```
Comment (SkySkimmer) [inline CHANGES]: Where?

Comment (psteckler) [inline CHANGES]: Some of the messages mentioned CoqIDE specifically, but those messages also apply to PG/xml or vsCoq. See PR #812.

Comment (maximedenes) [inline CHANGES]: I'll remove this change from the log if you don't mind, I don't think it brings much to the user to learn about it.

File: CHANGES (hunk 1 above)
Comment (maximedenes) [inline CHANGES]: Removed.

File: CHANGES (hunk 2 above)
Comment (maximedenes) [inline CHANGES]: Removed.

**Pull request merged**